MonoGame.Framework/Vector3.cs

```diff
@@ -778,12 +783,9 @@ public static void Normalize(ref Vector3 value, out Vector3 result)
         /// <returns>Reflected vector.</returns>
         public static Vector3 Reflect(Vector3 vector, Vector3 normal)
         {
-            // I is the original array
-            // N is the normal of the incident plane
-            // R = I - (2 * N * ( DotProduct[ I,N] ))
             Vector3 reflectedVector;
```

Comment:
Explanatory comments like these shouldn't be removed.  They help to explain what is happening in the following code.